contents/docs/product-analytics/dashboards.mdx

```diff
@@ -83,23 +83,17 @@ On paid plans, text cards can be added to dashboards alongside insights. These a
 
 On paid plans, tags can be added to dashboards to help better organize them. These tags can be added or removed by any member within a project.
 
-## Send or share a Dashboard
+## Sharing a dashboard
 
 Click 'Share' in the top right corner. You can:
 
--   Determine who within your project can access the dashboard
--   Create a link to share your dashboard publicly or embed your dashboard on a website
-
-### Advanced sharing
-
-<FeatureAvailability availability={_frontmatter.availability.features.advancedSharing} />
-
-On enterprise plans, you can further configure sharing to only give access to certain members within a project. Dashboards can be shared either by members with administrator privileges, or by the dashboard creator.
+-   With the Teams add-on, you can configure sharing to only give edit access to certain members within a project. Dashboards can be shared either by members with administrator privileges or by the dashboard creator.
```

Comment:
Can we hyperlink to the "Teams add-on" here? Doubt most people will know what it is